ms-frontend/src/services/reviewService.ts

```diff
@@ -19,7 +19,14 @@ export const reviewService = {
       }
     })
```

Comment:
`const response = await apiClient.pos(....

}
return new Review(response.data)
`

You need to add this to the createReview method too, sorry forgot to mention it.